### ISSUE TITLE
New install requires manual librairy installs  

### ISSUE THREAD

**Issue by ariane-blk** [AMYPAD/AmyPET#33]
New AmyPET set-up requires manual installs of numpy and packaging librairies (fails when installing dipy?).

**Comment by casperdcl** [comment]
yes I noticed this recently too in https://github.com/NiftyPET/NIMPA/blob/d0dc4f87f6d555a884301e74985117bed0d0e326/.github/workflows/test.yml#L50
